R/zchunk_L103.ghg_an_USA_S_T_Y.R

```diff
@@ -34,40 +36,69 @@ module_emissions_L103.ghg_an_USA_S_T_Y_DISABLED <- function(command, ...) {
     L107.an_Prod_Mt_R_C_Sys_Fd_Y <- get_data(all_data, "L107.an_Prod_Mt_R_C_Sys_Fd_Y")
     EPA_FCCC_AG_2005 <- get_data(all_data, "emissions/EPA_FCCC_AG_2005")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` and/or 'match' calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Merge-and-Match
-    # ===================================================
+    # Map EPA ghg emissions to GCAM sector and fuel,
+    # aggregate and convert from Gg to Tg
+
+    if(OLD_DATA_SYSTEM_BEHAVIOR) {
+    # Old system yields an emissions estimate of 0 for poultry because it has NA values.
+    L103.ghg_tg_USA_an_Sepa_F_2005 <- EPA_FCCC_AG_2005 %>%
+      left_join(EPA_ghg_tech, by = "Source_Category") %>%
```

Comment:
No, in this case there should be NAs. 
Will add comments. 
Will make corrections in other places to use `left_join_error_no_match` when applicable.